examples/path_whitelist/README.md

```diff
@@ -0,0 +1,34 @@
+Gohan path whitelist example
+--------------------
+
+In this example, we show how to define a list of paths that never
+require an authorization. A single path is defined as a regular
+expression (regex).
+
+In order to define a whitelisted path, add the following entry
+in the configuration file:
+
+``` yaml
+# path whitelist configuration
+path_whitelist:
```

Comment:
As I stated in previous post, `path_whitelist`, IMO isn't descriptive enough. Additionally, I don't believe this config should be in config root. Have you considered putting it in `route` section.

Actually, since we already have builtin Policy resource, it would make most sense from app developer perspective to simply put this information there. I understand, it may require additional refactor, so perhaps we should skip it for now. @nati , what do you think about it?